model/patch_lifecycle.go

```diff
@@ -446,6 +451,21 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 	return patchVersion, nil
 }
 
+func GetParameters(patchDoc *patch.Patch) error {
```

Comment:
I think we'd be better off with this as a method on the child patch. I think this way for two reasons:
1. when possible it helps to hang functions on structs to reduce clutter in the greater namespace (in this case the model package, which is pretty large)
2. It's unexpected for a function to modify its inputs. When you pass a patch into the function you wouldn't know that the patch will change without knowledge of the function. (Although it is a common idiom in other languages...)

Reply:
Good point, thank you!